crypto/nrf_cc310_platform/src/nrf_cc310_platform_mutex_freertos.c

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <stdint.h>
+#include <stddef.h>
+
+#include "FreeRTOS.h"
```

Comment:
Only use " includes when the path is relative from this file.

Reply:
This is not for zephyr usage. I don't control the include scheme in the outside project requiring this. system includes is not used in this case.